src/components/mediainfo/mediainfo.js

```diff
@@ -257,11 +256,17 @@ import '../../elements/emby-button/emby-button';
             if (item.Type === 'Audio') {
                 miscInfo.push(datetime.getDisplayRunningTime(item.RunTimeTicks));
             } else {
-                minutes = item.RunTimeTicks / 600000000;
-
-                minutes = minutes || 1;
-
-                miscInfo.push(`${Math.round(minutes)} mins`);
+                const totalMinutes = Math.round(item.RunTimeTicks / 600000000) || 1;
+                const totalHours = Math.floor(totalMinutes / 60);
+                const remainderMinutes = totalMinutes % 60;
+                const result = [];
+                if (totalHours > 0) {
+                    result.push(`${totalHours}h`);
+                }
+                if (totalMinutes > 0) {
```

Comment:
This is always `true` because `totalMinutes` is `1` if the rounding result is zero.

Reply:
I debated removing this already, so I'll just remove it lol